cd/publish_images.sh

```diff
@@ -51,16 +83,18 @@ source "${ROOT_DIR}"/docker/gaffer-pyspark-notebook/.env
 # JHUB_OPTIONS_SERVER_VERSION
 source "${ROOT_DIR}"/docker/gaffer-jhub-options-server/get-version.sh
 
-# Log in to Dockerhub
+# Log in to Docker Hub Container Image Library
 docker login -u "${DOCKER_USERNAME}" -p "${DOCKER_PASSWORD}"
+# Log in to GitHub Container Repository
+docker login ghcr.io -u "${GHCR_USERNAME}" -p "${GHCR_PASSWORD}"
 
-# Push images to Dockerhub
-pushTags gchq/hdfs "${HADOOP_VERSION}" "${APP_VERSION}"
-pushTags gchq/accumulo "${ACCUMULO_VERSION}" "${APP_VERSION}"
-pushTags gchq/gaffer "${GAFFER_VERSION}" "${APP_VERSION}"
-pushTags gchq/gaffer-rest "${GAFFER_VERSION}" "${APP_VERSION}"
-pushTags gchq/gaffer-ui "${GAFFER_TOOLS_VERSION}" "${APP_VERSION}"
-pushTags gchq/gaffer-road-traffic-loader "${GAFFER_VERSION}" "${APP_VERSION}"
-pushTags gchq/gaffer-pyspark-notebook "${GAFFER_VERSION}" "${APP_VERSION}"
-pushTags gchq/gaffer-jhub-options-server "${JHUB_OPTIONS_SERVER_VERSION}" "${APP_VERSION}"
-pushTags gchq/spark-py "${SPARK_VERSION}" "${APP_VERSION}"
+# Push to Container Repositories
+pushContainer gchq/hdfs "${HADOOP_VERSION}"
+pushContainer gchq/accumulo "${ACCUMULO_VERSION}"
+pushContainer gchq/gaffer "${GAFFER_VERSION}-accumulo-${ACCUMULO_VERSION}"
+pushContainer gchq/gaffer-rest "${GAFFER_VERSION}"
```

Comment:
I think gaffer-rest needs `ACCUMULO_VERSION` too

Reply:
Yes it does, missed that. Will fix it now.